tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py

```diff
@@ -230,63 +233,44 @@ def assert_all_weights_associated(weights, centroid_index):
     # Weights should now be all clustered with the centroid 1
     assert_all_weights_associated(l.layer.kernel, centroid_index=1)
 
-  def testClusterReassociation2(self):
-    """Verifies that the association of weights to cluster centroids are updated every iteration."""
-
+  def testSameWeightsAreReturnedBeforeAndAfterSerialisation(self):
+    """Verify that the weights of a cluster_wrapper are the same
+    before and after serialisation."""
     # Create a dummy layer for this test
     input_shape = (1, 2,)
-    l = cluster_wrapper.ClusterWeights(
-        keras.layers.Dense(8, input_shape=input_shape),
-        number_of_clusters=2,
-        cluster_centroids_init=CentroidInitialization.LINEAR
+    original_layer = cluster_wrapper.ClusterWeights(
```

Comment:
The test currently fails with this error at line 252:
AttributeError: 'ClusterWeights' object has no attribute 'outputs'

You may want to wrap this in a keras model for the test to pass.

Reply:
It looks like someone on your side had already fixed this yesterday: [https://github.com/tensorflow/model-optimization/blame/master/tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py#L246](https://github.com/tensorflow/model-optimization/blame/master/tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper_test.py#L246)
Should I still update this PR anyway?